SuperBass.java

```diff
@@ -8,7 +8,7 @@ public static void main(String[] args) {
         catch (InterruptedException ex)
         {
         }
-        System.out.println("This one is for the boys with the booming system") ;
+        System.out.println("This one is for the boys with the booming system");

```

Comment:
I was thinking something more like:
`a = "This one is ";
b = "for the boys ";
c = "with the booming system";
System.out.println(a + b + c);`